### ISSUE TITLE
CreateDate isn’t always a ExifDateTime object

### ISSUE THREAD

**Issue by Offirmo** [photostructure/exiftool-vendored.js#73]
First, thanks for this awesome library. The feature of auto-extracting the timezone is a killer! I can't imagine the amount of work required behind this lib.

**Describe the bug**
A date is not extracted as an `ExifDateTime` causing a crash when I try to call `.toDate()`

**To Reproduce**
My code works for my most of my pictures. However when processing this one:
![DSCN5363](https://user-images.githubusercontent.com/603503/75631991-d55a7800-5c4b-11ea-93d5-0a0868d8873d.JPG)

when trying to read the CreateDate: 
```ts
const date_object: undefined | ExifDateTime  = (exif_data as any)['CreateDate']
if (!date_object) return 
const tms = date_object.millis || +date_object.toDate()
```

I get a crash on calling `.toDate()`. Printing the "date_object", I get: `"0000:00:00 00:00:00"` which seems to be a string instead of an `ExifDateTime` object. 

**Expected behavior**
the CreateDate field should either be null or an ExifDateTime object

**Environment (please complete the following information):**
- npm version: exiftool-vendored@^9: version "9.7.0" (exiftool-vendored.exe@^11.86.0:
  version "11.86.0", exiftool-vendored.pl@^11.86.0: version "11.86.0")
 - OS and version: Mac OS X 10.14.6
 - Node.JS version: 12.16.1

**Comment by mceachen** [comment]
If date fields aren't parsable, the raw string from exiftool will be provided.

BTW, you're casting your tags to `any`, and then casting to something else, which causes all sorts of problems. Tag values are messy. You need to validate a field before blindly calling .toDate (or any other method).

**Comment by Offirmo** [comment]
> If date fields aren't parsable, the raw string from exiftool will be provided.

Then shouldn't the typings be updated to `CreateDate?: ExifDateTime | string` to reflect that?

I'm casting to any for simplicity, it's a prototype. Typing properly is tricky in my case.

**Comment by mceachen** [comment]
Oof, you’re completely right. I’ll update that soon.

**Comment by mceachen** [comment]
fixed in just-released v10.0.0
